spec/intro.dd

```diff
@@ -10,7 +12,7 @@ to create native libraries or executables.)
 $(P This document is the reference manual for the D Programming Language. For more information and
 other documents$(COMMA) see $(LINK2 https://dlang.org/, The D Language Website).)
 
-$(H2 Phases of Compilation)
+$(H2 $(LNAME2 Phases-of-Compilation, Phases of Compilation))
```

Comment:
Normally we use lowercase for anchors.

Reply:
Thank you. Fixed the anchor to lowercase.